src/test/kotlin/com/atlassian/performance/tools/virtualusers/ExploratoryVirtualUserTest.kt

```diff
@@ -23,12 +23,12 @@ class ExploratoryVirtualUserTest {
     @Test
     fun shouldRespectMaxLoad() {
```

Comment:
Locally the red commit fails nicely:
```
java.lang.AssertionError: 
Expecting:
 <4200.0 over PT1H>
to be between:
 [1800.0 over PT1H, 2400.0 over PT1H]
```
It shows that it is applying way too much load, just like we saw in actual erroring benchmarks.